packages/tabs/src/lib/factory.js

```diff
@@ -16,6 +16,7 @@ export default ({ node, settings }) => {
         settings,
         node,
         activeIndex: activeIndex !== undefined ? +activeIndex : +settings.activeIndex,
+        activeTabIndex: activeIndex !== undefined ? +activeIndex : +settings.activeIndex,

```

Comment:
Just for clarification, why does this not check/set activeTabIndex within the ternary?

Reply:
ActiveIndex is a property that can be passed in when you initialise a tab set - so that you could have tab 3 visible on page load, say.

So if the tab set is starting with panel 3 open, then the activeTab will also be 3. This saves having to pass in both values and potentially have them mis-match.  I should probably comment that though, as it does look like a blatant typo!

Reply:
#89e485dc9f7c71ab22c0d24d519d9537bfc17983

Reply:
Updated with comment in [https://github.com/stormid/components/pull/186/commits/89e485dc9f7c71ab22c0d24d519d9537bfc17983](https://github.com/stormid/components/pull/186/commits/89e485dc9f7c71ab22c0d24d519d9537bfc17983)